winml/test/concurrency/ConcurrencyTests.cpp

```diff
@@ -253,11 +253,11 @@ void MultiThreadMultiSessionOnDevice(const LearningModelDevice& device) {
 }
 
 void MultiThreadMultiSession() {
-    MultiThreadMultiSessionOnDevice(LearningModelDeviceKind::Cpu);
+    MultiThreadMultiSessionOnDevice(LearningModelDevice(LearningModelDeviceKind::Cpu));
```

Comment:
Not sure why these were needed, in ORT it's calling the correct constructor for `LearningModelDevice(const LearningModelDeviceKind&)`, while in the WAI build it must be given explicitly... Maybe some mismatching compiler options